sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -118,7 +119,9 @@ extern "C" {
  *          </a>.
  */
 typedef struct {
+#ifdef MODULE_GNRC_NDP_NODE
```

Comment:
I meant something that would explicitly remove this module from the used modules in a non-border router case.